src/main/java/com/coherent/training/selenium/stanila/pages/LoginPOM.java

```diff
@@ -1,12 +1,13 @@
 package com.coherent.training.selenium.stanila.pages;
-
-import com.coherent.training.selenium.stanila.Waits;
+import com.coherent.training.selenium.stanila.waits.Waits;
 import com.coherent.training.selenium.stanila.base.BaseClass;
 import lombok.SneakyThrows;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
+import java.time.Duration;
+
 public class LoginPOM extends BaseClass{
     @FindBy (xpath = "//button[starts-with(@class,'Button2')]")
     private WebElement LOGIN_CENTRAL_PAGE;
```

Comment:
why upper case? do you have more appropriate property for this element? not button2